includes/admin/stripe-settings.php

```diff
@@ -269,6 +269,17 @@
 				'large'   => __( 'Large (56px)', 'woocommerce-gateway-stripe' ),
 			],
 		],
+		'optimized_checkout_layout'           => [
+			'title'       => __( 'Optimized Checkout Layout', 'woocommerce-gateway-stripe' ),
+			'type'        => 'select',
+			'description' => __( 'Select the layout of the container.', 'woocommerce-gateway-stripe' ),
+			'default'     => 'accordion',
```

Comment:
We should use the constant here:
```suggestion
			'default'     => WC_Stripe_UPE_Payment_Gateway::OPTIMIZED_CHECKOUT_DEFAULT_LAYOUT,
```